static/bidder-info/kargo.yaml

```diff
@@ -13,4 +13,7 @@ userSync:
   redirect:
     url: "https://crb.kargo.com/api/v1/dsync/PrebidServer?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r={{.RedirectURL}}"
     userMacro: "$UID"
-endpointCompression: "GZIP"
\ No newline at end of file
+endpointCompression: "GZIP"
+openrtb:
+  gpp-supported: true
+  multiformat_supported: true
```

Comment:
[BidderInfo's openrtb struct](https://github.com/prebid/prebid-server/blob/df072ff78891c49a2e011dcb7dcb2f5f0d9c6d7f/config/bidderinfo.go#L98-L101) supports only `version` and `gpp-supported`. There is no support for multiformat_suported value. Please consider to remove it.